lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -106,7 +116,9 @@ export abstract class ClientApplication {
      *
      * @param configuration Object for the MSAL PublicClientApplication instance
      */
-    constructor(configuration: Configuration) {
+    constructor(operatingContext: StandardOperatingContext, configuration: Configuration) {
```

Comment:
I think @tnorling is suggesting we remove configuration from this API signature (it's new so not breaking) and call operatingContext.getConfig()... on line 129. Right?

Reply:
Right

Reply:
Yep.  I'll do this.